paddle/fluid/operators/controlflow/conditional_block_op.cc

```diff
@@ -56,7 +56,9 @@ class ConditionalBlockOp : public ConditionalOp {
 
     if (need_run) {
       auto *scope_var = scope.FindVar(Output(ConditionalOp::kScope));
-      PADDLE_ENFORCE(scope_var != nullptr, "Must set scope");
+      PADDLE_ENFORCE_NE(scope_var, nullptr,
+                        platform::errors::InvalidArgument(
+                            "Scope must be set in conditional_block_op"));
```

Comment:
May be better to use `PADDLE_ENFORCE_NOT_NULL`  and `platform::errors::PreconditionNotMet`.

Reply:
Done